### DISCUSSION TITLE
Get crash info from vgamepad

### DISCUSSION THREAD

**Discussion by Palamabron** [trackmania-rl/tmrl#64] [Q&A] [answered]
Hi, as it is impossible to get boolean info for crash detection from openplanet API, I noticed that Xbox controller shakes every time the collision occurs. Is it possible to get info for pad shaking from vgamepad?

**Comment by yannbouteiller** [comment, marked as answer]
As a matter of fact, on windows, it is possible :)

JumpyzZ has added this cool feature,  you can [register a callback](https://github.com/yannbouteiller/vgamepad#rumble-and-leds) that does whatever you want to handle the rumble event.

**Reply by Palamabron** [reply]
It works! Thanks!
